### PR TITLE
Replace squeeze/split with unpack

### PR DESCRIPTION
**Pull request by khryniewicz**
In [Recurrent Neural Networks](https://www.tensorflow.org/versions/r0.11/tutorials/recurrent/index.html) tutorial file `ptb_word_lm.py` I've replaced the alternative version of the code with `unpack` in place of `squeeze` and `split` (as in [documentation](https://www.tensorflow.org/versions/r0.11/api_docs/python/array_ops.html#split)).

### CODE REVIEW

File: tensorflow/models/rnn/ptb/ptb_word_lm.py [hunk 1]
```diff
@@ -130,8 +130,7 @@ def __init__(self, is_training, config, input_):
     #
     # The alternative version of the code below is:
     #
-    # inputs = [tf.squeeze(input_step, [1])
-    #           for input_step in tf.split(1, num_steps, inputs)]
+    # inputs = tf.unpack(inputs, num_steps, 1)
```
Comment (vrv) [inline tensorflow/models/rnn/ptb/ptb_word_lm.py]: I think there's an ongoing rename to move from unpack to unstack (to match numpy).  Can you make that change here?

File: tensorflow/models/rnn/ptb/ptb_word_lm.py [hunk 2]
```diff
@@ -130,8 +130,7 @@ def __init__(self, is_training, config, input_):
     #
     # The alternative version of the code below is:
     #
-    # inputs = [tf.squeeze(input_step, [1])
-    #           for input_step in tf.split(1, num_steps, inputs)]
+    # inputs = tf.unstack(inputs, num_steps, 1)
```
Comment (vrv) [inline tensorflow/models/rnn/ptb/ptb_word_lm.py]: One more suggestion: might be nice to use named arguments here for clarity:

num=num_steps, axis=1

**Review by vrv — APPROVED** [review]
Approval after the suggested change.
